tools/model_tools/src/openvino/model_zoo/_configuration.py

```diff
@@ -192,15 +234,38 @@ def load_models(models_root, args):
 
             check_composite_model_dir(composite_model_config.parent)
 
-            if composite_model_name in composite_models:
-                raise validation.DeserializationError(
-                    'Duplicate composite model name "{}"'.format(composite_model_name))
-            composite_models.append(composite_model_name)
+            with composite_model_config.open('rb') as config_file, \
+                validation.deserialization_context('In config "{}"'.format(composite_model_config)):
+
+                composite_model = yaml.safe_load(config_file)
+                model_stages = {}
+                for stage in sorted(composite_model_config.parent.glob('*/model.yml')):
```

Comment:
is it possible to have composition of model composition in future? :)
if yes, rglob usage will be preferable instead of glob

Reply:
Right now it is not allowed https://github.com/openvinotoolkit/open_model_zoo/blob/master/tools/model_tools/src/openvino/model_zoo/_configuration.py#L164. I think composition of composition models would require more effort than using rglob, probably it's better to do when we have one.

Reply:
pk, then keep as is